libcaf_test/caf/test/reporter.cpp

```diff
@@ -112,7 +116,32 @@ struct colorizing_iterator {
           out->put(c);
         break;
       case off_read_color:
-        mode = off_escape;
+        switch (c) {
+          case 'R':
+            mode = off_escape;
+            break;
+          case 'G':
+            mode = off_escape;
+            break;
+          case 'B':
+            mode = off_escape;
+            break;
+          case 'Y':
+            mode = off_escape;
+            break;
+          case 'M':
+            mode = off_escape;
+            break;
+          case 'C':
+            mode = off_escape;
+            break;
+          case '0':
+            mode = verbatim;
+            return;
+          default:
+            CAF_RAISE_ERROR("invalid color code");
+            break;
```

Comment:
All cases do the same thing. So either fall through, or just ignore everything but `0`:

```
          default:
            mode = off_escape;
            break;
          case '0':
            mode = verbatim;
```